ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -324,25 +346,62 @@
                                    (.setTimeout js/window (fn [] (scroll/scroll-to-id "route-calendar-anchor")) 150)))
                    :row-selected? #(= % selected-route)}
 
-      [{:name "Reitti" :width "30%"
+      [{:name "Reitti" :width "20%"
         :read (juxt :route-short-name :route-long-name)
         :col-style style-base/table-col-style-wrap
         :format (fn [[short long]]
                   (str short " " long))}
 
        ;; Show Reitti/Määränpää column only if it does affect on routes.
        (when (service-is-using-headsign route-hash-id-type)
-         {:name "Reitti/määränpää" :width "30%"
+         {:name "Reitti/määränpää" :width "23%"
           :read :trip-headsign
           :col-style style-base/table-col-style-wrap})
 
-       {:name "Tunnistetut muutosajankohdat (kpl)"
-        :width "40%"
+       {:name "Muutoksia (kpl)"
+        :width "10%"
         :read identity
         :format (fn [row]
                   (if (= :no-change (:change-type row))
-                    "Ei tulevia muutoksia"
-                    (:count row)))}]
+                    "0"
+                    (:count row)))}
+       {:name "Aikaa 1. muutokseen"
+        :width "15%"
+        :read :different-week-date
+        :format (fn [different-week-date]
+                  (if-not different-week-date
+                    [tv-change-icons/labeled-icon [ic/navigation-check] "Ei muutoksia"]
+                    [:span
+                     (str (time/days-until different-week-date) " pv")
+                     [:span (stylefy/use-style {:margin-left "5px"
```

Comment:
Did you notice there's `icon-labeled-icon` intended to be used with e.g. `ote.ui.icon_labeled`. That gives a relative margin between the icon and label.
What's needed here more is the `:color` overload, which could be done by extending `icon-labeled-icon` using a color from style definitions we have. The colours there in turn come from the UI specification, which give us the semantic name and color code.

Reply:
These are the same old codes that I removed when multiple changes were introduced in ui. 
We have bit of an issue with these icon - label thingies in this view. There isn't enough space to all of this information. So I leave it as it is because it works kind of.